### PR TITLE
Include missing name and description fields in generated pomfiles

### PR DESCRIPTION
**Pull request by paulerickson**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,7 +10,7 @@ plugins {
 subprojects {
 
     group = 'edu.wisc.my.restproxy'
-    version = '2.1.0'
+    version = '2.1.1'
```
Comment (timlevett) [inline build.gradle:13]: since we didn't close out 2.1.0 we could reuse 2.1.0 if you wanted, up to you.

**Comment by timlevett** [comment]
:+1:

**Comment by nblair** [comment]
:+1:

File: build.gradle (hunk 1 above)
Comment (paulerickson) [inline build.gradle:13]: That's true, although I think it's a Bad Idea to have two revisions in source control with the same version, as a general rule (snapshots being the exception), and numbers are pretty cheap anyway.
Onward and upward!

**Pull request merged**